sdk/communication/communication-phone-numbers/recordings/node/phonenumbersclient__area_codes_lists/recording_can_list_all_geographic_area_codes.json

```diff
@@ -8,27 +8,42 @@
         "Accept-Encoding": "gzip,deflate",
         "Authorization": "Sanitized",
         "Connection": "keep-alive",
-        "User-Agent": "azsdk-js-communication-phone-numbers/1.2.0-beta.4 core-rest-pipeline/1.10.2 Node/v16.17.0 OS/(x64-Linux-5.10.102.1-microsoft-standard-WSL2)",
+        "User-Agent": "azsdk-js-communication-phone-numbers/1.2.1 core-rest-pipeline/1.12.3 Node/v18.18.0 OS/(x64-Windows_NT-10.0.22621)",
         "x-ms-client-request-id": "sanitized",
         "x-ms-content-sha256": "47DEQpj8HBSa\u002B/TImW\u002B5JCeuQeRkm5NMpJWZG3hSuFU=",
-        "x-ms-date": "Tue, 31 Jan 2023 20:48:50 GMT"
+        "x-ms-date": "Mon, 13 Nov 2023 09:32:36 GMT",
+        "x-ms-useragent": "acs-test"
```

Comment:
Just to double check, did we notify phone numbers team about this? I mean, it shouldn't break anything, but we're changing slightly their tests as well.

Reply:
Good point. Thank you for brining that up. I have started the conversation with them: https://teams.microsoft.com/l/message/19:e36b2705de6a48c9a7e9e0418cc7c752@thread.skype/1700214575804?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1700214575804&teamName=Azure%20Communication%20Services&channelName=Phone%20Numbers%20SDK&createdTime=1700214575804